k9mail-library/src/main/java/com/fsck/k9/mail/transport/smtp/SmtpTransport.java

```diff
@@ -566,17 +572,38 @@ private void sendMessageTo(List<String> addresses, Message message)
         Address[] from = message.getFrom();
         try {
             String fromAddress = from[0].getAddress();
-            if (m8bitEncodingAllowed) {
-                executeCommand("MAIL FROM:<%s> BODY=8BITMIME", fromAddress);
+            if (pipeliningSupported) {
+                pipelinedCommand = new LinkedList<>();
```

Comment:
You are right we can do use it as local variable and pass to the functions who need it.